README.rst

```diff
@@ -167,6 +167,17 @@ From then on, one may build and run all tests by doing::
 
     make check
 
+Publishing to Output's vcpkg-registry
+-------
+
+1) Create a Release in GitHub:
+
+https://github.com/outputinc/lager/releases
+
+2) When build finishes, merge the corresponding PR into the registry:
+
+https://github.com/outputinc/vcpkg-registry
+
```

Comment:
Are there any other version numbers that need to be bumped in this repo?

Reply:
if the version comes from the tag we should be good. Not sure how this identifies itself / matters.

Reply:
Ok makes sense 👍